packages/plugins/src/plugins/diagnostic-fix.plugin.ts

```diff
@@ -59,20 +59,30 @@ export class DiagnosticFixPlugin extends Plugin {
       // TODO: User should be able to choose one of the fixes form this list in interactive mode
       const fix = fixes.shift()!;
 
-      for (const fileTextChange of fix.changes) {
-        let text = this.service.getFileText(fileTextChange.fileName);
+      if (isInstallPackageCommand(fix)) {
+        try {
+          await this.service.getLanguageService().applyCodeActionCommand(fix.commands[0]);
```

Comment:
Why only `[0]`?

I though about another loop before `for (const fileTextChange of fix.changes) {`

```
for (const command of fix.commands) {
   await this.service.getLanguageService().applyCodeActionCommand(command);
}

for (const fileTextChange of fix.changes) { 
  ...
}
```